build-system/scripts/clean_image_tags

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-[ -n "${BUILD_SYSTEM_DEBUG:-}" ] && set -x # conditionally trace
```

Comment:
any reason to rename this everywhere rather than just use the existing variable? 
Dont care really, no need to change.

Reply:
yeah agree